back/src/models/card.ts

```diff
@@ -0,0 +1,49 @@
+import mongoose, { Schema, model } from 'mongoose';
+import ICard from '../interfaces/ICard';
+
+const TaskSchema: Schema = new Schema({
+    _id: { type: mongoose.Schema.Types.ObjectId, auto: true },
```

Comment:
Why use mongoose.Schema.Types.ObjectId instead of Schema.Types.ObjectId? it was used in the CardSchema, did it not work here?